packages/std-client/src/setup/setupMUDV2Network.ts

```diff
@@ -178,6 +178,7 @@ export async function setupMUDV2Network<C extends ContractComponents>({
     components,
     singletonEntityId: SingletonID,
     singletonEntity,
+    /* @deprecated playerEntityId is equivalent to playerEntity */
```

Comment:
should we guide folks on which one to use?

Reply:
I'm not sure if the `deprecated` tag here shows up in vscode, but the intention was to deprecate `playerEntityId` and just use `playerEntity`